Application/Models/Users/AddUserModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
```

Comment:
The User models appear to be empty, please look into this

Reply:
Check out Ticket-Application-Feature3.  Work appears